From 6b082f2c1e76683688e8832c05246b2f91e18eab Mon Sep 17 00:00:00 2001
From: Ariel Aviad <aaviad@habana.ai>
Date: Tue, 08 Apr 2025 14:27:42 +0300
Subject: [PATCH] [SW-225120] g3: skyriver: crete serdes littlefs image

Update serdes littlefs image to include skyriver
serdes configuration
currently it holds unknown configuration and will
be change in the future.

Change-Id: I3a90c519f01e0799ab721261a9728381868584e1
Signed-off-by: Ariel Aviad <aaviad@habana.ai>
---

diff --git a/board/habanalabs/gaudi3/serdes/FlashMemory.bin b/board/habanalabs/gaudi3/serdes/FlashMemory.bin
index a75525a..488bd15 100644
--- a/board/habanalabs/gaudi3/serdes/FlashMemory.bin
+++ b/board/habanalabs/gaudi3/serdes/FlashMemory.bin
Binary files differ
